python/src/opendp/_lib.py

```diff
@@ -102,21 +102,26 @@ class DPNamespace(object):
         def __init__(self, expr):
             self.expr = expr
 
-        def laplace(self, scale=None):
-            """Add Laplace noise to the expression.
+        def noise(self, scale=None, distribution=None):
+            """Add noise to the expression.
 
             If scale is None it is filled by `global_scale` in :py:func:`opendp.measurement.make_private_lazyframe`.
 
-            :param scale: Noise scale parameter for the Laplace distribution. `scale` == standard_deviation / sqrt(2). 
+            :param scale: Noise scale parameter for the distribution.
             """
-            scale = float("nan") if scale is None else scale
             return pl.plugins.register_plugin_function(
                 plugin_path=lib_path,
-                function_name="laplace",
-                kwargs={"scale": scale},
+                function_name="noise",
+                kwargs={"scale": scale, "distribution": distribution},
                 args=self.expr,
                 is_elementwise=True,
             )
+        
+        def laplace(self, scale=None):
+            return self.noise(scale=scale, distribution="Laplace")
+        
+        def gaussian(self, scale=None):
+            return self.noise(scale=scale, distribution="Gaussian")
```

Comment:
Should these have doc strings, perhaps to explain what scale means? Usually we want people to make explicit choices, but is scale optional here? Where does the default come from?

Reply:
Scales are optional because we want the library to be able to set them for you to satisfy a given privacy level.